tests/integration/release/test_release.py

```diff
@@ -49,6 +48,25 @@ def test_row_noising_omit_row_or_do_not_respond(
     run_omit_row_or_do_not_respond_tests(dataset_name, config, original_data, noised_data)
 
 
+def test_column_dtypes(
```

Comment:
I've asked this multiple times but remind me again - how will we run this test as it previously existed (on sample data) but NOT during release-testing? i.e. we need to continue running the previous test every night like we currently are.

Reply:
The fixtures are set up to read in sample data if we don't run pytest with the --release flag.